src/components/navigation-header/components/give-feedback-button/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReactElement } from 'react'
+import classNames from 'classnames'
+import { IconMessageSquareFill24 } from '@hashicorp/flight-icons/svg-react/message-square-fill-24'
+import { IconExternalLink16 } from '@hashicorp/flight-icons/svg-react/external-link-16'
+import ButtonLink from 'components/button-link'
+import { useDeviceSize } from 'contexts'
+import s from './give-feedback-button.module.css'
+
+const FORM_URL = 'https://forms.gle/fnHLuNahLEhjuKvE6'
+
+function GiveFeedbackButton(): ReactElement {
+  const { isMobile } = useDeviceSize()
+
+  if (isMobile) {
+    return (
+      <a
+        href={FORM_URL}
+        target="_blank"
+        aria-label="Give beta feedback"
+        className={classNames(s.iconButton, s.primary)}
+        rel="noreferrer"
+      >
+        <IconMessageSquareFill24 />
+      </a>
+    )
+  } else {
+    return (
+      <ButtonLink
+        target="_blank"
+        href={FORM_URL}
+        text="Give beta feedback"
+        icon={<IconExternalLink16 />}
+        iconPosition="trailing"
+      />
+    )
+  }
```

Comment:
Can we set the ARIA label for both of these components to be `"Give beta feedback (opens in new tab)"`? That will give non-sighted screen reader users the affordance they need to know how the link behaves.

Reply:
✅ Absolutely - updated to `"Give beta feedback (opens in new tab)"` for both links 👍